torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -238,6 +238,17 @@ def sample_inputs_special_polygamma_n(n):
     return partial(sample_inputs_elementwise_njt_unary, op_kwargs={"n": n})
 
 
+def sample_inputs_masked_select(
+    op_info, device, dtype, requires_grad, op_kwargs=None, **kwargs
+):
+    for njt in _sample_njts(
+        device=device, dtype=dtype, requires_grad=requires_grad, dims=[1]
```

Comment:
Why is dims not 2 here?

Reply:
Oh, I guess  0, 1 or 2 all produce 2D NJTs here. Will swap to 2 for clarity